wayland-wm/sway/sway-9999.ebuild

```diff
@@ -15,35 +15,30 @@ EGIT_REPO_URI="https://github.com/SirCmpwn/sway.git
 LICENSE="MIT"
 SLOT="0"
 KEYWORDS=""
-IUSE=""
+IUSE='systemd'
 
-DEPEND="${RDEPEND}
+DEPEND="
+	${RDEPEND}
+	app-text/asciidoc
 	"
 
 RDEPEND="
-	app-text/asciidoc
 	dev-libs/json-c
 	dev-libs/libpcre
 	dev-libs/wlc
-	x11-base/xorg-server[wayland]"
-
+	x11-base/xorg-server[wayland]
+	x11-libs/cairo
+	x11-libs/libxkbcommon
+	x11-libs/pango
+	systemd? ( sys-apps/systemd )
+	"
 
 src_prepare() {
-	cmake-utils_src_prepare
-}
-
-src_configure() {
-	cmake-utils_src_configure
-}
-
-src_compile() {
-	cmake-utils_src_compile
-}
-
-src_test() {
-	cmake-utils_src_test
+	sed -i 's|set(CMAKE_C_FLAGS "-g")||' CMakeLists.txt || \
```

Comment:
```
sway[615]: Oct 25 2015 14:58:30 glibc-gentoo-hardened-check: *** buffer overflow detected ***: sway terminated; report to <http://bugs.gentoo.org/>
```

```
[23:17:08] <zetok> I might have found a way to reproduce the crash :3
[23:17:38] <SirCmpwn> that's good
[23:17:52] <zetok> SirCmpwn: https://github.com/zetok/zetok-overlay/pull/2/files#diff-9861a9110088b3f9beb23509dcb0dea9R37 ← when I removed that part from the ebuild, sway stopped crashing, adding it again caused it to crash again
[23:18:13] <SirCmpwn> well why did you remove -g
[23:18:17] <SirCmpwn> I left it there on purpose
[23:18:46] <zetok> 'twas not my doing
```

So, yeah, after that gets corrected, PR looks like good for merge, and sorry for the delay with the review `:)`